cpu/cpu_test.go

```diff
@@ -59,16 +59,27 @@ func TestTimes(t *testing.T) {
 }
 
 func TestCounts(t *testing.T) {
-	v, err := Counts(true)
+	logicalCount, err := Counts(true)
 	common.SkipIfNotImplementedErr(t, err)
-	require.NoError(t, err)
-	assert.NotZerof(t, v, "could not get logical CPU counts: %v", v)
-	t.Logf("logical cores: %d", v)
-	v, err = Counts(false)
+	if err != nil {
```

Comment:
Would you mind reverting it back to using the `testify/assert` or `testify/require` package? We've standardized on using `testify` in accordance with #1815, so it would be great to keep things consistent.